pkg/datarepair/datarepair_test.go

```diff
@@ -36,9 +44,11 @@ func TestDataRepair(t *testing.T) {
 		satellite.Repair.Checker.Loop.Pause()
 		satellite.Repair.Repairer.Loop.Pause()
 
-		testData := testrand.Bytes(1 * memory.MiB)
+		testData := make([]byte, 1*memory.MiB)
+		_, err := rand.Read(testData)
+		require.NoError(t, err)
```

Comment:
There's testrand package that makes this part more convenient.

Reply:
oh shoot my mistake. I can make a tiny pr fix if needed

Reply:
https://github.com/storj/storj/pull/2408